src/js/components/StarRating/StarRating.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Star, StarOutline } from 'grommet-icons';
+import { StarRatingPropTypes } from './propTypes';
+import { RadioButtonGroup } from '../RadioButtonGroup';
+
+const StarRating = ({ color, scale = 5, ...rest }) => {
+  const [rating, setRating] = useState();
```

Comment:
Can we call this "value" instead of "rating"? A soft opinion but it might make it clearer that this is an input.

Reply:
yes that aligns more with other areas I can change